api/v1alpha1/costmanagement_types.go

```diff
@@ -36,6 +36,15 @@ const (
 
 	// CertCheck allows certificate validation to occur.
 	CertCheck bool = true
+
+	//UploadOn sets the operator to upload to cloud.redhat.com
+	UploadOn bool = true
+
+	//UploadOff sets the operator to not upload to cloud.redhat.com
+	UploadOff bool = false
```

Comment:
do we need both of these?

Reply:
These are constants so I think its ok.

Reply:
Ahh ok, I understand now.

Reply:
We could use one and just do `!UploadOn` which is fine, but then let's do the same for the `CertIgnore`. I'm flexible either way. I think its about readability, but I can see also avoiding anyone ever mistakenly setting UploadOff as `true`.